src/core/server/http/http_server.test.ts

```diff
@@ -887,6 +888,53 @@ describe('conditional compression', () => {
       expect(response.header).not.toHaveProperty('content-encoding');
     });
   });
+
+  describe('response headers', () => {
+    it('allows to configure "keep-alive" header', async () => {
+      const { registerRouter, server: innerServer } = await server.setup({
+        ...config,
+        keepaliveTimeout: 100_000,
```

Comment:
(unrelated) I guess that's legacy stuff, but I wonder why the config value is in millisec when the `keep-alive` header only accepts a value in seconds.

Reply:
I don't know why it's still number, not `schema.duration` that uses human-readable format. it seems to have been overlooked.